libs/SalesforceSwiftSDK/SalesforceSwiftSDKTests/SFRestAPITests.swift

```diff
@@ -28,42 +28,14 @@ import SalesforceSDKCore
 import PromiseKit
 @testable import SalesforceSwiftSDK
 
-class SFRestAPITests: XCTestCase {
-    static var testCredentials: SFOAuthCredentials?
-    static var testConfig: TestConfig?
-    static var setupComplete = false
-    
+class SFRestAPITests: SalesforceSwiftSDKBaseTest {
+  
     override class func setUp() {
```

Comment:
Do we need this? Isn't `super.setUp()` implicit?

Reply:
@bhariharan  I intentionally left the override related code around due to addl. setup that may be required for individual test classes. I am not done with the tests, anticipate some setup/teardown related changes.